### ISSUE TITLE
Angular.Io Docs section "Updates and releases"=>"Upgrade to Angular latest version" points to version 16, not 17.

### ISSUE THREAD

**Issue by johanlindholm** [angular/angular#53649]
### Describe the problem that you experienced

Angular.Io Docs section "Updates and releases"=>"Upgrade to Angular latest version" points to version 16, not 17.

https://angular.io/guide/update-to-version-16

![angular-16](https://github.com/angular/angular/assets/6137590/c8e56713-a30d-451c-8396-4aed718018bc)

### Enter the URL of the topic with the problem

https://angular.io/guide/update-to-version-16

### Describe what you were looking for in the documentation

I was looking for information regarding updating to angular 17.

Before there was a link to your new "beta" page, but I do not see that link any more.

Klicking on the Angular update guide on the update-to-version-16 page you get directed to a site showing how to upgrade from 16 to 17.

![angular-16-17](https://github.com/angular/angular/assets/6137590/aec19c6e-912f-45e1-b617-e6b650ea4de4)

**Comment by mahalakshmi-veeraraj** [comment]
Hi All,

I am working on upgrading the Angular version from 12 to 16. 

Angular migration is failing if I missed specifying the --force with the ng update command.

ng update @angular/core@13 @angular/cli@13 --force.

I got a missing peer dependencies error. Hence, I added --force with the ng update command.

My doubt is: Is it a good practice to use --force with the ng command?

After the successful migration, I will need to deploy the changes on the server.

I will also need to execute --force on the server. Is it a good practice?

If it is not a good practice, then we need help with missing peer dependencies, Angular version migration issue solution.

If this is already fixed, kindly help me to get the details of it.

Thanks!
